src/BaseStrategy.js

```
@@ -0,0 +1,80 @@
+        this.logger = logger;
+        this.init();
+    }
+
+    /**
+     * Returns the instruction to ask the user follow instructions. This method must be implemented by all derived classes.
+     * @abstract
+     * @throws {ErrorWithMessage} When called directly on BaseStrategy
+     * @returns {void}
+     */
+    static getProviderInstruction() {
+        throw new ErrorWithMessage(`${this.strategy}: getProviderInstruction method is not implemented`);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)